proto/.clang-format

```diff
@@ -0,0 +1,8 @@
+---
+BasedOnStyle: Google
+IndentWidth: 4
```

Comment:
I personally like 2 better but 4 seems like the standard.

Reply:
I like 4, just because it lines up with go tabs better, but you're usually coming from ts which is 2 right?

Reply:
> ts which is 2 right?

we picked 2, I'm not sure if there is consensus on what's standard.

Reply:
lets go with 4